### PR TITLE
Add ARM build support

### PR DESCRIPTION
**Pull request by dz333n**
image
image
image
image

### CODE REVIEW

**Comment by m417z** [comment]
Interesting. Thanks for the pull request.

File: src/winspy.vcxproj [hunk 1]
```diff
@@ -75,7 +94,7 @@
     <Link>
       <SubSystem>Windows</SubSystem>
       <DelayLoadDLLs>uxtheme.dll;windowscodecs.dll</DelayLoadDLLs>
-      <AdditionalDependencies>psapi.lib;version.lib;uxtheme.lib;windowscodecs.lib;comctl32.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>psapi.lib;version.lib;uxtheme.lib;windowscodecs.lib;comctl32.lib;gdi32.lib;Advapi32.lib;Shell32.lib;Ole32.lib;%(AdditionalDependencies)</AdditionalDependencies>
```
Comment (m417z) [inline src/winspy.vcxproj:97]: Is this change necessary?

Comment (dz333n) [reply src/winspy.vcxproj:97]: I'm not sure. I compiled with VS 2017 and it complained me for some unresolved functions.
